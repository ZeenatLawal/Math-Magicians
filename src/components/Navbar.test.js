import React from 'react';
import { BrowserRouter as Router, NavLink } from 'react-router-dom';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const navbar = renderer
    .create(
      <Router>
        <NavLink to="/">Home</NavLink>
      </Router>
    )
    .toJSON();
  expect(navbar).toMatchSnapshot();
});

it('renders correctly', () => {
  const navbar = renderer
    .create(
      <Router>
        <NavLink to="/calculator">Calculator</NavLink>
      </Router>
    )
    .toJSON();
  expect(navbar).toMatchSnapshot();
});
