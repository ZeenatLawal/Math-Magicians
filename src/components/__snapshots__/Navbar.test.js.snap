// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  aria-current="page"
  className="active"
  href="/"
  onClick={[Function]}
  style={Object {}}
>
  Home
</a>
`;

exports[`renders correctly 2`] = `
<a
  aria-current={null}
  href="/calculator"
  onClick={[Function]}
>
  Calculator
</a>
`;
